devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy

```diff
@@ -198,7 +196,7 @@ private getButtonAttributesMap() {
 }
 
 private isEverspring() {
-    zwaveInfo.model.equals("0003")
+    return (zwaveInfo.model == "0003" ||  zwaveInfo.model == "D001")
```

Comment:
Putting here your own model doesn't seem right, after all it isn't `Evepsring`.

Reply:
OK, I am independent as a function.